### PR TITLE
Ignore timeout when waiting for bills

### PR DESCRIPTION
**Pull request by josepfo**

### CODE REVIEW

**Comment by josepfo** [comment]
[Ignore timeout when waiting for bills removed on f56](https://trello.com/c/c63qdEHt/620-ignore-timeout-when-waiting-for-bills-removed-on-f56)

File: lib/f56/f56-dispenser.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ BillDispenser.prototype.billsPresent = function billsPresent () {
 
 BillDispenser.prototype.waitForBillsRemoved = function waitForBillsRemoved () {
   return new Promise((resolve, reject) => {
+    var retries = 3
```
Comment (csrapr) [inline lib/f56/f56-dispenser.js:64]: I think using var is disallowed by the linter, but this is lamassu-machine so, not sure.

```suggestion
    let retries = 3
```

**Review by csrapr — CHANGES_REQUESTED** [review]

File: lib/f56/f56-dispenser.js (hunk 1 above)
Comment (josepfo) [reply lib/f56/f56-dispenser.js:64]: Hmm, I just checked and there's let/var in multiple files. Do you think it's worth changing?

Comment (csrapr) [reply lib/f56/f56-dispenser.js:64]: nah

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: lib/f56/f56-rs232.js [hunk 2]
```diff
@@ -134,6 +134,11 @@ function request (command) {
       rs232StatusPointer.off()
       statusPointer.off()
       if (status === 'Response') return resolve(frame)
+      if (status === 'Response Timeout') {
+        const error = new Error('Can\'t send in state: ' + dLevelFsm.state)
```
[the remark this answers is not in the document]
Comment (josepfo) [reply lib/f56/f56-rs232.js:138]: If the amount reaches 4 digits it goes to a new line. Should I increase the width even more?

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**